skycoin/skywire
# add `dmsgpty-cli` to `skywire-cli`

Did you run `make format && make check`? Yes

Fixes  #863

 Changes:	
- add `dmsgpty` command groups to `skywire-cli` as follow:
  - `skywire-cli dmsgpty list` to get list of all remote visor connected to hypervisor
  - `skywire-cli dmsgpty start <pk>` to start dmsgpty command line interface. Just note that if remote visor connection lost, then its appear on `skywire-cli dmsgpyt list` command yet, but when trying to start dmsgpty, get error after a while (about 10 second).

How to test this PR:
- run a hypervisor
- run a visor and set hypervisor PK to its config
- use `skywire-cli dmsgpty list` check that visor connected and available
- use `skywire-cli dmsgpty start <pk>` to run dmsgpty command line interface of remote visor in your terminal

ScreenCast:

https://user-images.githubusercontent.com/79150699/172659506-fc3e5e9c-ee46-4352-8bc5-076f338c4c9f.mp4
